package.json

```diff
@@ -48,6 +48,17 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "d3": "^4.2.2",
+    "d3-array": "^1.0.1",
```

Comment:
There is no need to manually import all d3 modules, as the entry point in the `d3` module already has references to all d3 modules.